js/jest-tests/src/ScoreFrame.js

```diff
@@ -0,0 +1,76 @@
+export const STRIKE = 'X';
+export const SPARE = '/';
+export const MISS = '-';
+
+export const MAX_POINTS = 10;
+export const MIN_POINTS = 0;
+
+export const MAX_TURN_TO_ADD_ROLLS = 8;
+export const STRIKE_ROLLS = 2;
+export const SPARE_ROLLS = 1;
+export const NO_ROLLS = 0;
+
+export class ScoreFrame {
+
+  constructor (roll_one, roll_two, roll_three, turn) {
```

Comment:
Para discusión: a lóxica de contas queda bastante clara pero a verdade é que temos 9 frames normais e un raro. Quizá facendo un ScoreFrame con 2 rools e un ScoreLastFrame con hasta rolls quedaría unha lóxica máis simple para os 9 primeiros frames (ou 8 se tes en conta o bonus)